src/models/packet_macros.rs

```diff
@@ -68,7 +68,9 @@ macro_rules! packet_boilerplate {
             byte_vec.extend(size_vec);
 
             //Send the packet
-            stream.write_all(&byte_vec).unwrap();
+            stream.write_all(&byte_vec).unwrap_or_else(|e| {
+                warn!("Failed to write packet: {:?}", e);
+            });
```

Comment:
Changing this from a panic to being tolerant for the potential race condition. Can't think of a scenario where the closure of the stream in the first place wouldn't prevent this